tests/cli/test_project.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+from pathlib import Path
+
+from .conftest import working_directory
+
+
+def test_project_init(tmp_path: Path) -> None:
+    with working_directory(tmp_path):
```

Comment:
Please drop the test.
If you want to test something -- a check that mocked `cookiecutter.main.cookiecutter()` is called with the proper argument should be enough.
The test is not unit but integration, it fails if github connection is unstable (how often we encounted this last year?)

Also, the test fails on the template update, e.g. adding a license, etc. The client project should not care about the generated project structure, it is a responsibility of the template project.
Add these tests there if you want.

Reply:
The note is still valid

Reply:
OK, left uncovered.